testdata/README.md

```diff
@@ -1,22 +0,0 @@
-# Test data
```

Comment:
Why has this directory been removed? (I don't see why "generating the JSON schema" and "removing test data" would be in the same pull request in general, and I question whether it's a good idea. I use that test data in google-cloudevents-dotnet.)

Reply:
Sorry, this shouldn't have been removed. It would be out of scope anyways if removed.
Reverted.